todo_list_project/tasks/tests.py

```
@@ -326,27 +331,37 @@ def setUp(self):
+    def test_task_lifecycle_multilingual(self):
         """
-        Test the complete lifecycle of a task.
+        Test the complete lifecycle of a task in multiple languages.
 
         This test simulates creating, updating, completing, and deleting a task,
-        verifying each step of the process.
+        verifying each step of the process in both English and Spanish.
+        """
+        for lang_code in ['en', 'es']:
+            with self.subTest(lang=lang_code):
+                with override_settings(LANGUAGE_CODE=lang_code):
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)